### DISCUSSION TITLE
Blazin BF mix

### DISCUSSION THREAD

**Discussion by Szymonkruk12** [FunkinCrew/Funkin#3481] [Ideas]
Please add it because i wanna see Boyfriend fighting with Darnell

**Comment by boyfriendfnflel** [comment]
that'd honestly be pretty cool, ngl

**Comment by ChillyBeanBAM** [comment]
[It's been confirmed to be coming in the future.](https://x.com/News_Funkin/status/1858351897637265601)

**Comment by Beelijah** [comment]
heck yeah i love this
